app/support/data_mapper_setup.rb

```diff
@@ -0,0 +1,6 @@
+require 'data_mapper'
+require './app/models/user.rb'
+require './app/models/peep.rb'
+
+DataMapper.setup(:default, ENV['DATABASE_URL'] || "postgres://localhost/chitter_andy_#{ENV['RACK_ENV']}")
```

Comment:
Metrics/LineLength: Line is too long. [105/90] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)